packages/data-models/appConfig.ts

```diff
@@ -200,6 +200,14 @@ const ASSET_PACKS = {
   folderName: "asset_packs",
 };
 
+const TASKS = {
+  enabled: true,
+  // The name of the field that tracks the currently highlighted task group
+  highlightedTaskField: "_task_highlighted_group_id",
```

Comment:
nit(non-blocking)
Given the properties already sit under `Tasks` can probably get rid of the work `task` from `highlightedTaskField` and `taskGroupsListName` property. Perhaps something like `activeGroupField`

Reply:
Discussed in [this WIP RFC](https://docs.google.com/document/d/1qVcUri2DAhyiFx6ihtFLeVeJjoS1ba0rdxntRg3AGec/edit).

Reply:
Linked explicitly with comment:
https://docs.google.com/document/d/1qVcUri2DAhyiFx6ihtFLeVeJjoS1ba0rdxntRg3AGec/edit?disco=AAABC5ZxhSw

Will resolve here for now